### DISCUSSION TITLE
有誰也會用高八度的聲音跟家裡的寵物講話

### DISCUSSION THREAD

**Discussion by a3240281370** [1101-GS4719-C/Python-Community-GS4719-C#1409] [General]
不知道大家跟自己的寵物或路上的貓狗講話時，聲音會不會跟原本的不一樣呢，我發現好像很多人都會換另一種聲音耶～​
最近看到一個很有趣的實驗研究，實驗請十位女性以超drama的裝可愛娃娃音對狗狗花式吹捧，比如「怎摸拿摸Q窩喔」、「這有一隻超可i的狗勾」、「Good~Boy~」，然後再以正常的口吻重複一次這些台詞
​而當科學家將這些溢美之詞播放給不同年齡的狗狗聽後，竟發現只有一歲以下的小狗狗會豎起耳朵、或轉頭注視聲音來源，至於那些已經長大的狗狗則愛理不理，牠們在聽到正常說話時，反應反而更大，證明不只是人類，就連長大的狗狗，也不太想鳥裝可愛的聲音。
原來用這種方式講話的時候，狗狗都是用鄙視的眼神在看我嗎哈哈哈

**Comment by pigeon9975** [comment]
我之前養倉鼠的時候雖然不太會用高八度的聲音講話，但我跟他講話時會變得很喜歡疊字字，感覺我們只要面對寵物就會把他當成小朋友，就算以動物的年齡來說他們已經七老八十了，但我們還是會本能性的把他們當小孩看，好吧，看來我活該被鄙視qq

**Comment by PaulSu0905** [comment]
我之前看過一個研究對婦女使用誇張的可愛的嬰兒談話和正常的講話來讚美狗進行了實驗。令人驚訝的是，只有一歲以下的小狗表現出興趣，而成年狗對正常的講話更感興趣，這表明它們可能不欣賞可愛的聲音。

**Comment by Miinteaa** [comment]
我會用高八度的聲音跟我家的倉鼠講話哈哈哈哈哈哈哈😂而且還很喜歡問他問題，明明他也不會發出叫聲回應，但就是會一直問🤣不知道有沒有人跟我一樣看到可愛的動物也會一直問他們問題的⋯（我知道聽起來很奇怪）

**Comment by zhiron544** [comment]
我覺得這篇文章很有趣，讓我想起我家的狗狗，牠也不喜歡我用娃娃音跟牠說話，每次都會轉頭走開。我覺得人類應該要多了解狗狗的喜好和個性，不要以為牠們都像小孩一樣需要被寵愛。狗狗也有自己的尊嚴和情感，我們應該要尊重牠們，用正常的語氣和態度跟牠們交流。
